src/async-wrap.cc

```diff
@@ -333,12 +333,7 @@ void PromiseWrap::GetParentId(Local<String> property,
 
 static void PromiseHook(PromiseHookType type, Local<Promise> promise,
                         Local<Value> parent, void* arg) {
-  Local<Context> context = promise->CreationContext();
-  Environment* env = Environment::GetCurrent(context);
-
-  // PromiseHook() should never be called if no hooks have been enabled.
-  CHECK_GT(env->async_hooks()->fields()[AsyncHooks::kTotals], 0);
```

Comment:
@trevnorris I know why you put it there, and I am not a *fan* of removing it, but do you have a better suggestion?

It’s good the cctest is there, it basically checks the same thing, right?